### DISCUSSION TITLE
IP Tables and Killswitch

### DISCUSSION THREAD

**Discussion by Hawkins1212** [bubuntux/nordvpn#235] [Q&A]
So first, I'll apologize in advance.  Somewhat of a amateur at linux and trying to understand the NordVPN Killswitch/iptables feature within the docker. 

I am running Unraid but can certainly have access and ability to use console.  My question is surrounding the killswitch.  I have two other containers/dockers that are being routed through this nordvpn docker.  I've verified it to be set up correctly using `curl ifconfig.io`     How do I ensure that if something happens to the Nordvpn docker, that it "kills" these other two dockers?  I understand this done via iptables but cannot find a good reference online with how to set this up.  TIA -- Hawkins

**Comment by bubuntux** [comment]
you could stop the vpn service using 
`nordvpn disconnect` inside the nordvpn container
then check the connection in the dependent containers
